### PR TITLE
Throw some words on there

### PR DESCRIPTION
**Pull request by naomi-lgbt**
I added some more thematic words to increase the variety. 🙂

### CODE REVIEW

File: lib/do_username.rb [hunk 1]
```diff
@@ -1,12 +1,12 @@
 module DOUsername
-  SEA_CREATURES = %w(walrus seal fish shark clam coral whale crab lobster starfish eel dolphin squid jellyfish ray giantSquid shrimp mantaRay angler snorkler scubaDiver).freeze
-  SEA_OBJECTS = %w(boat ship submarine yacht dinghy raft kelp).freeze
+  SEA_CREATURES = %w(walrus seal fish shark clam coral whale crab lobster starfish eel dolphin squid jellyfish ray giantSquid shrimp mantaRay angler snorkler scubaDiver urchin anemone morel axolotl).freeze
```
Comment (MattIPv4) [inline lib/do_username.rb]:
```suggestion
  SEA_CREATURES = %w(walrus seal fish shark clam coral whale crab lobster starfish eel dolphin squid jellyfish ray shrimp mantaRay angler snorkler scubaDiver urchin anemone morel axolotl).freeze
```
As giant is being added to the size descriptors, we can just remove the giant variant of squid here

**Review by MattIPv4 — CHANGES_REQUESTED** [review]
`descriptors += SIZE_DESCRIPTORS unless noun.start_with?('giant')`
⬇️
`descriptors += SIZE_DESCRIPTORS`

As we're removing `giantSquid`, we can remove this check

File: lib/do_username.rb (hunk 1 above)
Comment (naomi-lgbt) [reply lib/do_username.rb]: No more GiantGiantSquid 😛

File: lib/do_username.rb [hunk 2]
```diff
@@ -45,7 +45,7 @@ def random_noun
 
   def random_descriptor(noun)
     descriptors = DESCRIPTORS.dup
```
Comment (MattIPv4) [inline lib/do_username.rb]:
```suggestion
    descriptors = DESCRIPTORS + SIZE_DESCRIPTORS
```
Can combine

File: lib/do_username.rb [hunk 3]
```diff
@@ -45,7 +45,7 @@ def random_noun
...
-    descriptors += SIZE_DESCRIPTORS unless noun.start_with?('giant')
+    descriptors += SIZE_DESCRIPTORS
```
Comment (MattIPv4) [inline lib/do_username.rb]:
```suggestion
```
Combined with previous line

**Review by MattIPv4 — CHANGES_REQUESTED** [review]

File: lib/do_username.rb [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
-  SEA_OBJECTS = %w(boat ship submarine yacht dinghy raft kelp).freeze
+  SEA_CREATURES = %w(walrus seal fish shark clam coral whale crab lobster starfish eel dolphin squid jellyfish ray shrimp mantaRay angler snorkler scubaDiver urchin anemone morel axolotl).freeze
+  SEA_OBJECTS = %w(boat ship submarine yacht dinghy raft kelp seaweed anchor).freeze
 
-  DESCRIPTORS = %w(cute adorable lovable happy sandy).freeze
-  SIZE_DESCRIPTORS = %w(large big small).freeze
-  CREATURE_DESCRIPTORS= %w(swimming).freeze
+  DESCRIPTORS = %w(cute adorable lovable happy sandy bubbly friendly).freeze
+  SIZE_DESCRIPTORS = %w(large big small giant massive tiny little).freeze
+  CREATURE_DESCRIPTORS= %w(swimming floating drifting sleeping eating hiding).freeze
```
Comment (MattIPv4) [inline lib/do_username.rb]: I wonder if `floating` and `drifting` should move into `DESCRIPTORS`, as I think all `SEA_OBJECTS` can also float/drift?

Comment (naomi-lgbt) [reply lib/do_username.rb]: Everything except the anchor, I think. 😅

Comment (naomi-lgbt) [reply lib/do_username.rb]: I moved them. :)

**Review by MattIPv4 — APPROVED** [review]
Lgtm

**Pull request merged**